.github/workflows/skunk.yml

```diff
@@ -0,0 +1,54 @@
+name: Skunk
+on: [push]
```

Comment:
We want to also add the `pull_request` event so CI status is reflected in the PR

`on:[push, pull_request]`